help/status-and-availability.md

```diff
@@ -84,13 +87,9 @@ and compose box. Status emoji and status messages are also shown on
 
 {tab|mobile}
 
-Access this feature by following the web app instructions in your
-mobile device browser.
-
-Implementation of this feature in the mobile app is tracked [on
-GitHub](https://github.com/zulip/zulip-flutter/issues/197). If
-you're interested in this feature, please react to the issue's
-description with 👍.
+1. Tap on a user's profile picture or name to view their status message if
+   they have one set. Their status message appears under their name on
+   their profile.
```

Comment:
Maybe the second sentence should be a new paragraph within this bullet? (You do this rather than a paragraph outside by intending it 3 spaces). 

Just to make the simple instruction more visibly simple.

Reply:
Updated to:

<img width="764" height="280" alt="Screenshot 2025-07-10 at 21 43 25" src="https://github.com/user-attachments/assets/9fa20c9c-03a6-40d1-964c-a96277e515e2" />